src/msgpack/FloatConverter.cs

```diff
@@ -1,114 +1,116 @@
 using System;
-using System.IO;
 using System.Runtime.InteropServices;
 
 namespace TarantoolDnx.MsgPack
 {
     internal class FloatConverter : IMsgPackConverter<float>, IMsgPackConverter<double>
     {
-        public void Write(double value, Stream stream, MsgPackContext context)
+        public void Write(double value, IMsgPackWriter writer, MsgPackContext context)
         {
             var binary = new DoubleBinary(value);
-            stream.WriteByte((byte) DataTypes.Double);
+            writer.Write(DataTypes.Double);
             if (BitConverter.IsLittleEndian)
             {
-                stream.WriteByte(binary.byte7);
-                stream.WriteByte(binary.byte6);
-                stream.WriteByte(binary.byte5);
-                stream.WriteByte(binary.byte4);
-                stream.WriteByte(binary.byte3);
-                stream.WriteByte(binary.byte2);
-                stream.WriteByte(binary.byte1);
-                stream.WriteByte(binary.byte0);
+                writer.Write(binary.byte7);
```

Comment:
This will lead to an additional allocation. Maybe bad.

Need perfomance testing.